packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/index.tsx

```diff
@@ -66,7 +66,9 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
   identityVerificationActions: bindActionCreators(
     actions.components.identityVerification,
     dispatch
-  )
+  ),
+  analyticsActions: bindActionCreators(actions.analytics, dispatch),
+  isEddQualified: true
```

Comment:
hardcoded value ok???

Reply:
yeah this PR is still WIP since we miss BE :(